developer_manual/digging_deeper/config/appconfig.rst

```diff
@@ -0,0 +1,152 @@
+=========
+AppConfig
```

Comment:
Shouldn't this go into https://github.com/nextcloud/documentation/blob/845fb9a79fe360b762690e735fcab236f9c16898/developer_manual/basics/storage/configuration.rst? There are now two pages for configuration

Reply:
Yes, migrate the new content into the old page